raiden-dapp/src/scss/colors.scss

```diff
@@ -1,17 +1,18 @@
 // Please also check plugins/vuetify.ts for theme colors
-$primary-color: #28A5C8;
+$primary-color: #28a5c8;
```

Comment:
Any specific reason the colors are changed to lower case from upper case? I can see we are not using them consistently across the app but it feels like something that should be enforced.

Reply:
This I have not reflected over at all. :)

Reply:
Hmm, I believe I've been copying colors back and forth from illustrator or photoshop. They are in lowercase there by default.

Reply:
Actually not, they get automatically set to lower case when saving the file. Maybe it is a VSCode setting. Frustrating...